### PR TITLE
feat: cli: print beneficiary info in state miner-info

### PR DESCRIPTION
**Pull request by geoff-vball**
## Checklist

Before you mark the PR ready for review, please make sure that:
- [ ] All commits have a clear commit message.
- [ ] The PR title is in the form of of `<PR type>: <area>: <change being made>`
    - example: ` fix: mempool: Introduce a cache for valid signatures`
    - `PR type`: _fix_, _feat_, _INTERFACE BREAKING CHANGE_, _CONSENSUS BREAKING_, _build_, _chore_, _ci_, _docs_,_perf_, _refactor_, _revert_, _style_, _test_
    - `area`: _api_, _chain_, _state_, _vm_, _data transfer_, _market_, _mempool_, _message_, _block production_, _multisig_, _networking_, _paychan_, _proving_, _sealing_, _wallet_, _deps_
- [ ] This PR has tests for new functionality or change in behaviour
- [ ] If new user-facing features are introduced, clear usage guidelines and / or documentation updates should be included in https://lotus.filecoin.io or [Discussion Tutorials.](https://github.com/filecoin-project/lotus/discussions/categories/tutorials)
- [ ] CI is green

### CODE REVIEW

File: cmd/lotus-miner/info.go [hunk 1]
```diff
@@ -313,6 +313,21 @@ func handleMiningInfo(ctx context.Context, cctx *cli.Context, fullapi v1api.Full
 	}
 	colorTokenAmount("Total Spendable:  %s\n", spendable)
 
+	fmt.Println()
+	fmt.Printf("Beneficiary:\t%s\n", mi.Beneficiary)
```
Comment (magik6k) [inline cmd/lotus-miner/info.go:317]: Wouldn't print this when there's no beneficiary

**Review by arajasek — APPROVED** [review]

**Comment by jennijuju** [comment]
Would loved some comment in code that it’s goanna be supported nv17 and up

**Comment by geoff-vball** [comment]
@jennijuju In nv16 it won't print anything at all (so same behaviour until upgrade).

**Comment by jennijuju** [comment]
Yup I saw that in the code. Tho code comment is not for users, but developers. Any developer clone the lotus master today would thought this  (beneficiary) is suppose to be a thing while it’s not, yet.

**Pull request merged**